code/game/objects/items.dm

```diff
@@ -795,3 +799,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 /obj/item/proc/doStrip(mob/stripper, mob/owner)
 	return owner.dropItemToGround(src)
+
+///adds a random pixel offset, for non-precise item placement such as throwing or dropping to the ground
+/obj/item/proc/pixelOffset()

```

Comment:
this should be a define imo

and also include random in the name

Reply:
I thought about this but I'm not sure where to put it, is on top of this file okay?

Reply:
I don't see why it would have to be a `#define`

Reply:
@SpaceManiac it doesn't really make much sense as as a proc either, it's two lines

Reply:
just inline it.

Reply:
Why not set it to atom movable?

Reply:
This really shouldn't be a proc or define at all